internal/js/modules/k6/browser/browser/page_mapping.go

```diff
@@ -749,11 +757,15 @@ func parseWaitForFunctionArgs(
 	return js, popts, exportArgs(gargs), nil
 }
 
-func parseStringOrRegex(v sobek.Value) string {
+func parseStringOrRegex(v sobek.Value, doubleQuote bool) string {
 	var a string
 	switch v.ExportType() {
 	case reflect.TypeOf(string("")):
-		a = fmt.Sprintf("'%s'", v.String()) // Strings require quotes
+		if doubleQuote {
+			a = fmt.Sprintf(`"%s"`, v.String()) // Strings require quotes
+		} else {
+			a = fmt.Sprintf("'%s'", v.String()) // Strings require quotes
+		}
```

Comment:
Nit
```suggestion
		// Strings require quotes
		if doubleQuote {
			a = fmt.Sprintf(`"%s"`, v.String())
		} else {
			a = fmt.Sprintf("'%s'", v.String())
		}
```

Reply:
Done in 92d686ec922fb09639f039ee5d10d2a5d22b9c33